scripts/inc.packagetarget.sh

```diff
@@ -55,13 +55,13 @@ aromaupdatebinary() {
   copy "$SCRIPTS/aroma-resources/update-binary" "$build/META-INF/com/google/android/update-binary"
 }
 
-bundlexz() {
-  case "$ARCH" in #Include our own 32-bit xz-decompression binary
-    arm*) xzbin="xzdec-arm";;
-    x86*) xzbin="xzdec-x86";;
+bundlebusybox() {
+  case "$ARCH" in #Include official busybox binary
+    arm*) busyboxbin="busybox-armv7l";;
+    x86*) busyboxbin="busybox-i686";;
   esac
-  copy "$SCRIPTS/xz-resources/$xzbin" "$build/xzdec"
-  EXTRACTFILES="$EXTRACTFILES xzdec"
+  copy "$SCRIPTS/busybox-resources/$busyboxbin" "$build/busybox"
```

Comment:
I removed the `chmod +x "$build/busybox"`, as I have marked the busybox executable in git.